presto-cli/pom.xml

```diff
@@ -170,6 +183,96 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>rpm-maven-plugin</artifactId>
+                <version>2.2.0</version>
+                <executions>
+                    <execution>
+                        <id>generate-rpm</id>
+                        <goals>
+                            <goal>rpm</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <name>${project.name}</name>
```

Comment:
I changed this from `presto-cli-rpm` (which is the current convention/coincidence in presto-server-rpm module) to module name since this makes it easier to pass the final RPM name to enforcer plugin and to integration test.
The other alternative is to use something like `${project.name}-rpm`.

What do others think?